docs/sources/tempo/traceql/_index.md

```diff
@@ -101,7 +101,7 @@ possible to span-level intrinsics.
 
 ### Attribute fields
 
-There are two types of attributes: span attributes and resource attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
+There are three types of attributes: span attributes, resource attributes, and event attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
```

Comment:
I'm not sure about this one: What do you think about introducing the concept of scopes here instead of describing them as "attribute types"?
```suggestion
TraceQL has three different attribute scopes: span attributes, resource attributes, and event attributes. By expanding a span in the Grafana UI, you can see both its span attributes (1 in the screenshot) and resource attributes (2 in the screenshot).
```